conf/application.conf

```diff
@@ -50,6 +50,16 @@ dev-hub-base-url = "http://localhost:9680"
 
 platform.frontend.host = "http://localhost:9619"
 
+urls {
+  footer {
+    govukHelp = "https://www.gov.uk/help"
+    termsConditions = s"${host}/api-documentation/docs/help/terms-and-conditions"
+    privacy = "${host}/api-documentation/docs/help/privacy"
+    cookies = "${host}/api-documentation/docs/help/cookies"
+    accessibility = "https://www.tax.service.gov.uk/accessibility-statement/hmrc-developer-hub?referrerUrl=%2Fapi-test-user%2Fuser&_ga=2.85606041.1657921576.1600767148-457607700.1600072762"
```

Comment:
I don't think we want to include the _ga query param

Reply:
Shouldn't this look more like the master version?
`@{applicationConfig.platformFrontendHost}/accessibility-statement/hmrc-developer-hub?referrerUrl=@{helper.urlEncode(request.uri)}`